assets/stylesheets/layout/_main.scss

```diff
@@ -280,6 +280,13 @@ html.iframed {
 	line-height: 1;
 }
 
+/* Emoji and smileys */
+.emoji,
```

Comment:
I think this _is_ the right place and we can drop the styles elsewhere. I hope. :)